setup.py

```diff
@@ -121,5 +152,8 @@ def build_extensions(self):
     ext_modules=ext_modules,
     cmdclass={'build_ext': BuildExt},
     packages=find_packages(),
+    package_data = {
+        'brainiak.fcma': ['*.pyx', '*.pyxbld'],
+    },
```

Comment:
The pyx files corresponding to extensions are included by default. We do not use pyxbld files anymore. Please remove `package_data`.